pkg/operator/ceph/object/bucket/provisioner.go

```diff
@@ -154,7 +154,7 @@ func (p Provisioner) Grant(options *apibkt.BucketOptions) (*bktv1alpha1.ObjectBu
 	statement := NewPolicyStatement().
 		WithSID(p.cephUserName).
 		ForPrincipals(p.cephUserName).
-		ForResources(p.bucketName).
+		ForResources(p.bucketName, fmt.Sprintf("%s/*", p.bucketName)).
```

Comment:
Maybe a `ForSubresources` or `ForObjects()` method be a good addition?  It'd make for a cleaner input and would allow for a default value.

Reply:
Done